pkg/soc/soc.go

```diff
@@ -66,13 +66,28 @@ func (s *SOC) WrappedChunk() swarm.Chunk {
 
 // Chunk returns the SOC chunk.
 func (s *SOC) Chunk() (swarm.Chunk, error) {
-	socAddress, err := s.address()
+	socAddress, err := s.Address()
 	if err != nil {
 		return nil, err
 	}
 	return swarm.NewChunk(socAddress, s.toBytes()), nil
 }
 
+// Signature returns the SOC signature.
+func (s *SOC) Signature() []byte {
+	return s.signature
+}
+
+// OwnerAddress returns the ethereum address of the SOC owner.
+func (s *SOC) OwnerAddress() []byte {
+	return s.owner
+}
+
+// ID returns the SOC id.
+func (s *SOC) ID() []byte {
+	return s.id
+}
```

Comment:
why not public?

Reply:
Why not public fields? I didn't want to introduce big refactoring change in this PR, so I just exported existing methods. I will look in this, and potentially addressing this in separate PR.